plugins/woocommerce/client/admin/client/settings-payments/offline/settings-payments-bacs.tsx

```diff
@@ -0,0 +1,251 @@
+			] );
+			createSuccessNotice(
+				__( 'Settings updated successfully', 'woocommerce' )
+			);
+		} catch ( error ) {
+			createErrorNotice(
+				__( 'Failed to update settings', 'woocommerce' )
+			);
+		}
+	};
+
+	return (
+		<PaymentSettingsLayout>
+			<GatewaySettingsForm>
+				<PaymentSettingsSection
+					title={ __( 'Enable and customise', 'woocommerce' ) }
+					description={ __(
+						'Choose how you want to present bank transfer to your customers during checkout.',
+						'woocommerce'
+					) }
+				>
+					{ isLoading ? (
+						<FieldPlaceholder size="small" />
+					) : (
+						<CheckboxControl
+							label={ __(
+								'Enable direct bank transfers',
+								'woocommerce'
+							) }
+							checked={ Boolean( formValues.enabled ) }
+							onChange={ ( checked ) => {
+								setFormValues( {
+									...formValues,
+									enabled: checked,
+								} );
+							} }
+						/>
+					) }
+					{ isLoading ? (
+						<FieldPlaceholder size="medium" />
+					) : (
+						<TextControl
+							label={ __( 'Title', 'woocommerce' ) }
+							help={ __(
+								'This controls the title which the user sees during checkout.',
+								'woocommerce'
+							) }
+							placeholder={ __(
+								'Direct bank transfer payments',
+								'woocommerce'
+							) }
+							value={ String( formValues.title ) }
+							onChange={ ( value ) => {
+								setFormValues( {
+									...formValues,
+									title: value,
+								} );
+							} }
+						/>
+					) }
+					{ isLoading ? (
+						<FieldPlaceholder size="large" />
+					) : (
+						<TextareaControl
+							label={ __( 'Description', 'woocommerce' ) }
+							help={ __(
+								'Payment method description that the customer will see on your checkout.',
+								'woocommerce'
+							) }
+							value={ String( formValues.description ) }
+							onChange={ ( value ) => {
+								setFormValues( {
+									...formValues,
+									description: value,
+								} );
+							} }
+						/>
+					) }
+					{ isLoading ? (
+						<FieldPlaceholder size="large" />
+					) : (
+						<TextareaControl
+							label={ __( 'Instructions', 'woocommerce' ) }
+							help={ __(
+								'Instructions that will be added to the thank you page and emails.',
+								'woocommerce'
+							) }
+							value={ String( formValues.instructions ) }
+							onChange={ ( value ) => {
+								setFormValues( {
+									...formValues,
+									instructions: value,
+								} );
+							} }
+						/>
+					) }
+				</PaymentSettingsSection>
+
+				<PaymentSettingsSection
+					title={ __( 'Account details', 'woocommerce' ) }
+					description={ __(
+						'Configure your bank account details.',
+						'woocommerce'
+					) }
+				>
+					{ isLoadingAccounts ? (
+						<FieldPlaceholder size="large" />
+					) : (
+						<BankAccountsList
+							accounts={ accounts }
+							onChange={ setAccounts }
+							defaultCountry={ storeCountryCode }
+						/>
+					) }
+				</PaymentSettingsSection>
```

Comment:
```suggestion
				</PaymentGatewaySettingsSection>
```
I suggest changing this and according styling - so it's more specific, otherwise folks can this it's related to the payment settings page

Reply:
Removed in 370a44a843242f748568e6b9369d4d13cad5559b